config/jsdoc/api/template/static/scripts/main.js

```diff
@@ -108,4 +113,9 @@ $(function () {
 
     $(window).on('hashchange', _onHashChange);
     _onHashChange();
+
+    // Make links that JSDoc forgot clickable
+    $('p').each(function(i, p) {
+      p.innerHTML = anchorme(p.innerHTML);
+    });
```

Comment:
Is this a JSDoc bug or something we can undo later?

Reply:
Actually it's a bug, not a feature:
https://github.com/jsdoc3/jsdoc/blob/2c47d4b/lib/jsdoc/util/markdown.js#L59-L67

However, I'll create a follow-up pull request to handle this at doc creation time instead.

Reply:
Follow-up pull request #8852.